cpp/tests/io/csv_test.cu

```diff
@@ -110,6 +221,36 @@ void expect_column_data_equal(std::vector<T> const& lhs, cudf::column_view const
   EXPECT_THAT(cudf::test::to_host<T>(rhs).first, ::testing::ElementsAreArray(lhs));
 }
 
+void write_csv_helper(std::string const& filename,
+                      cudf::table_view const& table,
+                      bool include_header,
+                      std::vector<std::string> const& names = {})
+{
+  // NOTE sink_info, na, and metadata lifetimes must extend past write_csv call since
```

Comment:
The issue with `sink_info` and `na` was fixed

Reply:
Done.